tests/index.ts

```diff
@@ -0,0 +1,6 @@
+import { before } from 'node:test';
+import { configureServer } from '@src/server.config.js';
+
+before(() => {
+  configureServer();
+});
```

Comment:
```suggestion
before(async () => {
  await configureServer();
});
```

Importante pra que haja garantia de que o banco conectou e que o server rodou antes de começar os testes.